modules/network-observability-multitenancy.adoc

```diff
@@ -3,22 +3,43 @@
 // network_observability/installing-operators.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="network-observability-multi-tenancy{context}"]
+[id="network-observability-multi-tenancy_{context}"]
 = Enabling multi-tenancy in Network Observability
-Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project admins. Project admins who have limited access to some namespaces can access flows for only those namespaces.
+Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project administrators. Project administrators who have limited access to some namespaces can access flows for only those namespaces. 
+
+For Developers, multi-tenancy is available for both Loki and Prometheus but requires different access rights.
 
 .Prerequisite
 * You have installed at least link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7]
```

Comment:
Does the prereq to install Loki is still needed? If they are using prom mode then they dont need to install Loki

Reply:
Hmmmm maybe it needs to be updated. To "If you are using Loki, you have installed at least ..."